docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -26,361 +26,245 @@ Recommended configuration: 384kB Ram + 8MB SPIRAM + 4MB Flash
-This second "robot" can be as simple as an instance of `viam-server` running on your development machine.
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+. $HOME/esp/esp-idf/export-rs.sh
+```
 
-### Configure the ESP32 as a Remote
+Similarly to ESP-IDF to avoid conflicts with other toolchains, adding this command to your `.bashrc` or `.zshrc` is not recommended.
+Save this command to run in any future terminal session where you need to activate the ESP Rust development framework.
 
-Navigate to [the Viam app](https://app.viam.com).
-Create and configure a second robot as the controller robot.
+## Install the Micro-RDK
 
-Add the ESP32-backed worker robot as a remote of your new controller robot to establish a secure connection between both robots:
+To install the Micro-RDK on your ESP32 board:
 
-<div style="max-width:800px;"><img src="../../img/esp32-setup/esp32-remote-creation.png" alt="Adding the ESP32 as a remote in the Viam app Config builder." ></div>
+- [create a new robot in the Viam app](#create-a-new-robot)
+- [generate a new project from Viam's micro-RDK template](#generate-a-new-project-from-the-micro-rdk-template)
+- [upload the project to your ESP32](#upload-and-connect-to-your-esp32-board)
 
-1. Navigate to the **Code sample** tab of the worker robot, click on **Remotes**, and copy the code generated.
+### Create a New Robot
 
-   ![The remotes config information](../../img/microcontroller/copy-remotes-code.png)
+Navigate to [the Viam app](https://app.viam.com) and create a new robot in your desired location.
+Keep your `Mode` and `Architecture` selections at default, and skip the instructions in the **Setup** tab for now as the setup instructions there are not for microcontrollers.
 
-   {{% snippet "show-secret.md" %}}
+### Generate a New Project from the Micro-RDK Template
 
-2. Navigate to the **Config** tab of the control robot, select the `Remotes` subtab, select the `JSON` mode and click **Create Remote**.
+Use [the micro-RDK template](https://github.com/viamrobotics/micro-rdk-template.git) to create a new micro-RDK project to upload to your ESP32 by running:
 
-   ![Create remote menu](../../img/microcontroller/create-remote.png)
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+cargo generate --git https://github.com/viamrobotics/micro-rdk-template.git
+```
 
-3. Paste the copied `JSON` config into the configuration field.
+If you would like, you can use `mkdir` to initialize a new repository in the directory you created by running `cargo-generate`, to track any changes you make to the generated project.
 
-   ![Empty configuration field](../../img/microcontroller/config-field.png)
+You will be prompted to paste your Viam robot configuration information (`viam.json`) into the terminal.

```

Comment:
```suggestion
You will be prompted to paste your robot's JSON configuration into the terminal.
```
Opt but I think the viam.json note might be less helpful w the info immediately following

Reply:
👍